pkgs/applications/editors/vscode/default.nix

```diff
@@ -82,6 +82,11 @@ in
         --set-rpath "${rpath}" \
         $out/lib/vscode/resources/app/node_modules.asar.unpacked/keytar/build/Release/keytar.node
 
+      patchelf \
+        --set-rpath "${rpath}" \
+        "$out/lib/vscode/resources/app/node_modules.asar.unpacked/native-keymap/build/Release/\
+      keymapping.node"
```

Comment:
Does this need a backport?

Reply:
It seems that way, the error occurs also with 1.23.0 and disappears after this change. I made another PR against the release branch, #44055.